src/extension/serializer.ts

```diff
@@ -964,9 +1002,9 @@ export class GrpcSerializer extends SerializerBase {
     return outputs
   }
 
-  private static marshalCellExecutionSummary(
+  protected static marshalCellExecutionSummary(
```

Comment:
Why is this method being changed from private to protected? Where does it get invoked from?

Reply:
the plan was to add method signatures in the base to ensure they got implemented in children.  I can switch back to private or add the method signature. LMK.